src/user/user.service.spec.ts

```diff
@@ -1,18 +1,127 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { UserService } from './user.service';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { faker } from '@faker-js/faker';
 
 describe('UserService', () => {
-  let service: UserService;
+  let userService: UserService;
+
+  const mockUser = {
+    id: expect.any(Number),
+    email: expect.any(String),
+    password: expect.any(String),
+    role: expect.any(String),
+  };
```

Comment:
hmm, this does not look like a mock for me - more like a payload for an `expect` function

mock should have some values, e.g.:

```suggestion
  const mockUser = {
    id: 1234,
    email: faker.internet.email(),
    password: faker.internet.password(),
    role: 'USER'
  };
```

what you have here are instructions what values to expect in `expect` chain